src/main/java/hunternif/mc/impl/atlas/core/PlayerEventHandler.java

```diff
@@ -26,6 +26,7 @@ public static void onPlayerLogin(ServerPlayer player) {
 
     public static void onPlayerTick(Player player) {
         if (!AntiqueAtlasMod.CONFIG.itemNeeded) {
+        	// TODO Can we move world scanning to the server in this case as well?
```

Comment:
I think it is fine, because the per-player atlas is just a normal atlas with a special id.